.github/labeler.yml

```diff
@@ -174,7 +174,7 @@ integration:stackit:
       - any-glob-to-any-file: "integrations/stackit/**/*"
       - any-glob-to-any-file: ".github/workflows/stackit.yml"
 
-integration:together_ai:
+integration:together-ai:
```

Comment:
I think the label is indeed `integration:together_ai` right?

Reply:
sorry I think I'm misunderstanding. Should the label actually be `integration:together-ai`?

Reply:
sounds good!